### DISCUSSION TITLE
Code: 4000, Text = mutation conflicts, concurrent mutation detected while committing segment compaction operation

### DISCUSSION THREAD

**Discussion by lonesoul** [databendlabs/databend#11773] [Q&A]
replace into 
Code: 4000, Text = mutation conflicts, concurrent mutation detected while committing segment compaction operation
Databend DatabendQuery v1.1.56-draft

**Comment by volongoto** [comment]
when I use Update statement，It's show me the same error.  Maybe the feature of  databend  segment .
